pkgs/development/ocaml-modules/hacl-star/raw.nix

```diff
@@ -0,0 +1,51 @@
+{ lib, which, stdenv, fetchzip, ocaml, findlib, hacl-star, ctypes, cppo }:
+
+stdenv.mkDerivation rec {
+  pname = "ocaml${ocaml.version}-hacl-star-raw";
+  version = "0.3.2";
+
+  src = fetchzip {
```

Comment:
I guess you can use `fetchurl` here.

Reply:
Can't get it to work with `fetchurl` now, `fetchzip` works and I think you suggested that in the past?